hermes-management/src/main/java/pl/allegro/tech/hermes/management/domain/topic/TopicService.java

```diff
@@ -168,6 +176,10 @@ public String fetchSingleMessageFromPrimary(String brokersClusterName, TopicName
                 .collect(Collectors.toList());
     }
 
+    private List<Topic> listTopics(String groupName) {
```

Comment:
IMHO this should be left public, just as was before - same with groups. In our APIs we're not using it now, but let's treat this code as a feature of a library class one could use somewhere. :>